PSReadLine.build.ps1

```diff
@@ -139,46 +157,48 @@ $buildTestParams = @{
     Outputs = "TestPSReadLine/bin/$Configuration/TestPSReadLine.exe"
 }
 
-<#
-Synopsis: Build executable for interactive testing/development
-#>
-task BuildTestHost @buildTestParams BuildMainModule, {
-    exec { msbuild TestPSReadLine/TestPSReadLine.csproj /t:Rebuild /p:Configuration=$Configuration /p:Platform=AnyCPU }
-}
-
-
-$buildUnitTestParams = @{
-    Inputs  = { Get-ChildItem test/*.cs, test/PSReadLine.Tests.csproj }
-    Outputs = "test/bin/$Configuration/PSReadLine.Tests.dll"
-}
-
-
-<#
-Synopsis: Build the unit tests
-#>
-task BuildTests @buildUnitTestParams BuildMainModule, {
-    exec { msbuild test/PSReadLine.tests.csproj /t:Rebuild /p:Configuration=$Configuration /p:Platform=AnyCPU }
-}
-
 
 <#
 Synopsis: Run the unit tests
 #>
-task RunTests BuildTests, {
-    exec {
-        $env:PSREADLINE_TESTRUN = 1
-        $runner = "$PSScriptRoot\PSReadLine\packages\xunit.runner.console.2.3.1\tools\net452\xunit.console.exe"
-        if ($env:APPVEYOR)
-        {
-            $outXml = "$PSScriptRoot\xunit-results.xml"
-            & $runner $PSScriptRoot\test\bin\$Configuration\PSReadLine.Tests.dll -appveyor -xml $outXml
-            $wc = New-Object 'System.Net.WebClient'
-            $wc.UploadFile("https://ci.appveyor.com/api/testresults/xunit/$($env:APPVEYOR_JOB_ID)", $outXml)
-        }
-        else
+task RunTests {
```

Comment:
You removed `BuildTests` - please put in the dependency so one can run `Invoke-Build Clean,RunTests` .